packages/svelte/src/compiler/phases/2-analyze/visitors/Attribute.js

```diff
@@ -30,6 +30,10 @@ export function Attribute(node, context) {
 		}
 	}
 
+	if (node.name === 'autofocus' || node.name === 'muted') {
```

Comment:
Maybe you can use https://github.com/sveltejs/svelte/pull/14289/files#diff-14febff536dc266e984a0227108522a940c12342884fdf4a38b270d265b85d87 so that we don't have to keep track of these names in multiple places

Reply:
Can you hoist that change out of that PR and and put it up separately maybe? Then I can use it in this PR and the other one.

Reply:
done https://github.com/sveltejs/svelte/pull/14372